.docker/nginx/opendor.me.conf

```diff
@@ -0,0 +1,29 @@
+upstream opendor-app {
+   server app:9000;
+}
+
+server {
+    listen 80;
+
+    index index.php;
+
+    error_log  /var/www/storage/nginx/error.log;
```

Comment:
I believe it'd be better if we used `/dev/stderr` for both the error and access logs.

Reply:
I know you have things in the make file to properly create the logs. in my experience it's just easier if services in a container output to STDERR. That way for Docker on Mac users they see the nginx logs in the stream of other logging info like:

![Screen Shot 2021-07-16 at 11 15 27 AM](https://user-images.githubusercontent.com/619938/125970206-2e1c3a82-64c6-41a0-b4d9-0bfb43ce12e5.png)

Reply:
Okay, but in that case I would use `stderr` for the error log only and `stdout` for the access log. But reasoning sounds valid. Super strange that nginx can't just create the path on its own, that's what I expected.^^

Reply:
please use `make build` to create files and build image.

Two things -

- @Gummibeer You asked for these changes so right now it works as expected.
- @mallardduck What is the benefit of using `/dev/stderr` when you can already see logs in docker desktop .. output is always streamed.

Reply:
For me the current setup is fine!

Reply:
Using `make` wasn't an option for my testing due to the varying folder paths.

However I also don't follow you here:
> you can already see logs in docker desktop .. output is always streamed.

With the error log set to:
```
error_log  /var/www/storage/nginx/error.log;
```
Then errors encountered when starting Nginx will go to the docker log/stderr.
However we have to tail the local log file after Nginx starts logging to error.log (i.e. after it loads the config).

Same goes for the access log too - plus it is creating local logging data I don't need taking disk space.
However if we opt to use `/dev/stderr` then it's in then we're not using data and can still track the errors.

Reply:
Per the other thread - I've removed this change from the PR I sent.
All good.